assets/terraform-modules/bootkube/assets.tf

```diff
@@ -113,6 +113,7 @@ locals {
     cluster_dns_service_ip = cidrhost(var.service_cidr, 10)
     cluster_domain_suffix  = var.cluster_domain_suffix
     enable_tls_bootstrap   = var.enable_tls_bootstrap
+    cloud_provider         = var.cloud_provider
```

Comment:
This variable is not defined in `bootkube` module.

Reply:
I think it is https://github.com/kinvolk/lokomotive/blob/master/assets/terraform-modules/bootkube/variables.tf#L40